modules/apps/knowledge-base/knowledge-base-web/src/main/resources/META-INF/resources/admin/info_panel.jsp

```diff
@@ -215,6 +215,32 @@ if (ListUtil.isEmpty(kbFolders) && ListUtil.isEmpty(kbArticles)) {
 						<dd class="sidebar-dd">
 							<%= dateFormatDateTime.format(kbArticle.getModifiedDate()) %>
 						</dd>
+						<dt class="sidebar-dt">
+							<liferay-ui:message key="expiration-date" />
+						</dt>
+						<dd class="sidebar-dd">
+							<c:choose>
+								<c:when test="<%= kbArticle.getExpirationDate() != null %>">
+									<liferay-ui:message arguments="<%= new Object[] {dateFormatDateTime.format(kbArticle.getExpirationDate()), HtmlUtil.escape(kbArticle.getUserName())} %>" key="x-by-x" translateArguments="<%= false %>" />
+								</c:when>
+								<c:otherwise>
+									<liferay-ui:message key="never-expire" />
+								</c:otherwise>
+							</c:choose>
+						</dd>
+						<dt class="sidebar-dt">
+							<liferay-ui:message key="review-date" />
+						</dt>
+						<dd class="sidebar-dd">
+							<c:choose>
+								<c:when test="<%= kbArticle.getReviewDate() != null %>">
+									<liferay-ui:message arguments="<%= new Object[] {dateFormatDateTime.format(kbArticle.getReviewDate()), HtmlUtil.escape(kbArticle.getUserName())} %>" key="x-by-x" translateArguments="<%= false %>" />
+								</c:when>
+								<c:otherwise>
+									<liferay-ui:message key="never-review" />
+								</c:otherwise>
+							</c:choose>
+						</dd>
```

Comment:
Is it OK to show this information if feature flag is not enabled?

Reply:
yeah, totally, I forgot FF there.